### DISCUSSION TITLE
Currently running 0.15.0-849d441

### DISCUSSION THREAD

**Discussion by mjalindahl** [blakeblackshear/frigate#14129] [Ask A Question] [answered]
The new notifications are fantastic.  I get them before my HA blueprint ones, by a half second, but still counts.  My only issue is, are there plans for pictures/thumbnails/GIFs?

**Comment by hawkeye217** [comment]
Browser support is limited, unfortunately.

**Comment by NickM-27** [comment, marked as answer]
The images are already supported and implemented, they work on Android. All other browser platforms have not implemented it but when they do it will work without changes needed on Frigate side

See https://developer.mozilla.org/en-US/docs/Web/API/Notification/image
